src/components/APIurl.js

```diff
@@ -0,0 +1,5 @@
+// export const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/cKFfxRYPps1kHshp1Jyn';
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented our un unused and unwanted lines of code from your code base. It makes it look messy and cluttered.